protocol/error.go

```diff
@@ -26,8 +34,11 @@ const (
 	ErrorCouldNotVerify
 )
 
-// ErrorResponses contains error codes that
-// a response can omit the DirectoryResponse.
+// ErrorResponses contains the server responses indicating that
+// a client request could not be processed due to a
+// malformed client request or an internal server error.
+// Server response messages containing one of these errors therefore
+// omit the DirectoryResponse.
```

Comment:
I would like to change this to "ErrorResponses contains error codes indicating the client should omit the consistency checks. These errors indicate that either a client request could not be processed due to a malformed client request, an internal server error or due to a malformed server response." to make thing a bit clearer. 
Actually `ErrorResponses` is currently being used in the client side to omit the consistency check (it should include `ErrorMalformedDirectoryMessage` as well), and I think we should have a better name for this. What do you think?